CONTRIBUTING.md

```diff
@@ -18,6 +18,11 @@ Let us know you're interested in working on a feature by filing a ticket in our
 * After making your changes, be sure to run the [Blacklight tests](https://github.com/projectblacklight/blacklight/wiki/testing) to make sure everything works.
 * Submit your change as a [Pull Request](https://help.github.com/en/articles/about-pull-requests).
 
+## Pull Requests and Code Review
+- Pull requests must be reviewed and approved by one non-author with write-permissions
```

Comment:
```suggestion
- Pull requests must be reviewed and approved by someone who isn't the author
```

Reply:
I'd prefer to keep this language. See: https://github.com/projectblacklight/blacklight/pull/2116#issuecomment-506525886

Reply:
Works for me, I do appreciate the clarity. 👍